src/metabase/query_analysis.clj

```diff
@@ -74,8 +78,8 @@
   [query-type]
   (and (public-settings/query-analysis-enabled)
        (case query-type
-         :native (public-settings/sql-parsing-enabled)
-         :query true
+         :native     (public-settings/sql-parsing-enabled)
+         :query      true
```

Comment:
Interesting that cljfmt is happy both before and after this change, that's a pity. IntelliJ doesn't have an option to match this alignment, so we can easily churn on it. It certainly is more readable this way though.

Reply:
Yeah, it seems like most of the codebase is favoring the "align right-hand expressions" style. Surely there's a plugin for IntelliJ?